### ISSUE TITLE
Debug: start without debugging should also announce "Started running without debugging"

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#193790]
1. Launch Visual Studio code, create a sample .NET application with the below code
```
Console.WriteLine("Hello, World!");
Console.Read();
```
 

2. Build the application.
3. Start NVDA screen reader.
4. In VS Code, start the app without debugging by hit "Ctrl +F5"

Expected output

==============

On hitting Ctrl+ F5, screen reader should have said something like "Started running without debugging".  So, possibly the debugging toolbar  has the same header value set for Ctrl +F5 AND F5.

 

Actual output

==============

Screen Reader is speaking "Debugging Started"

**Comment by connor4312** [comment]
Can you please confirm the version of VS Code that you're using?

**Comment by connor4312** [comment]
Thanks, what language are you trying to debug?

**Comment by lunasfacilitator** [comment]
Python 3.10

**Comment by lunasfacilitator** [comment]
Just updated:

Version: 1.85.0 (user setup)
Commit: af28b32d7e553898b2a91af498b1fb666fdebe0c
Date: 2023-12-06T20:48:09.019Z
Electron: 25.9.7
ElectronBuildId: 25551756
Chromium: 114.0.5735.289
Node.js: 18.15.0
V8: 11.4.183.29-electron.0
OS: Windows_NT x64 10.0.19044

So, when I run with ctrl+f5 it says "debugging stopped" when it finishes. If I were to stop the run in the middle, it says the same thing.
